docs/docs/reference/crd-reference/analysis.md

```diff
@@ -4,10 +4,10 @@ comments: true
 
 # Analysis
 
-An `Analysis` resource customizes the templates
-that are defined in an
+An `Analysis` resource is an instance of an
```

Comment:
Maybe before this a sentence  on the line of "An Analysis  is a snapshot of your current application status, based on your defined SLIs  it can show if your SLO are satisfied... "

Reply:
done